### PR TITLE
Fix issue template so it isn't terrible

### PR DESCRIPTION
**Pull request by Geczy**
**What kind of change does this PR introduce?**
Refactor

**Summary**
Please understand `<!-- text -->` that this syntax allows COMMENTS that show to just the user, and does not show it in the issue so we don't have to read it. The pull request template is already using it and it is great. 

Made the command for output one command so user doesn't have to run multiple commands

**Does this PR introduce a breaking change?**
No

### CODE REVIEW

**Comment by Geczy** [comment]
No comment? Just closed? @deltaxflux

**Comment by deltaxflux** [comment]
I already sad my opinion about that changes.

**Comment by Geczy** [comment]
Why are you adding horizontal lines after every

---

question

--- 

in the issue template

Can you explain that one?

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,38 +1,17 @@
...
-
-
-#### Fluxion version:
-*Paste the output of* `cat fluxion.sh | grep revision`
-
-#### Linux version:
-*Paste the output of* `cat /proc/version` or `uname -r` 
+## Specifications
+<!-- Paste the output of:
+	grep -E 'FLUXIONVersion=|FLUXIONRevision=' fluxion.sh && cat /proc/version && airmon-ng && aireplay-ng wlan0 -9
```
Comment (Geczy) [inline .github/ISSUE_TEMPLATE.md]: @deltaxflux you should add this too because it is one command to get all versions, driver, chipset, etc

**Comment by deltaxflux** [comment]
Does make sense. Thanks

**Comment by Geczy** [comment]
You're welcome

Can you next explain why you choose not to use comment tag? `<!-- text -->`?

**Comment by deltaxflux** [comment]
I personally like it more but maybe I redesign the hole template later. (and change this)

**Pull request closed without merging**